.github/workflows/ansible-ci.yml

```
@@ -24,7 +24,7 @@ jobs:
           path: lint-rules
 
       - name: ansible-lint
-        uses: ansible/ansible-lint-action@master
+        uses: ansible/ansible-lint-action@main
```

ansible-lint action has been replaced by https://github.com/marketplace/actions/run-ansible-lint:

```suggestion
        uses: ansible/ansible-lint@main
```